src/components/EventList/index.js

```diff
@@ -4,10 +4,31 @@ import PropTypes from 'prop-types';
 import moment from 'moment';
 import Table from '../Table';
 
+const eventDateComparator = (eventA, eventB, isStart) => {
```

Comment:
a cleaner way of doing this would probably be to pass in the field that you're comparing on since you can access elements of objects as such

`object["key"] `

Reply:
The linter changed object["key"] to object.key so hopefully you're okay with that

Reply:
yeah ok - this is fine, just merge